packages/unmock-hash/src/hash/v0/index.ts

```diff
@@ -181,7 +181,7 @@ type ActionsActionsV0 =
   | "make-header-keys-lowercase"
   | "deserialize-json-body"
   | "deserialize-x-www-form-urlencoded-body";
-type ActionsV0 = ActionsActionsV0 | ActionsActionsV0[];
+export type ActionsV0 = ActionsActionsV0 | ActionsActionsV0[];
```

Comment:
Re the above comment above clarity of types, concretely I would suggest dropping `ActionsV0`, renaming `ActionsActionsV0` to `ActionsV0` and just being specific about it where needed (accepting either a list of it or a single argument, or perhaps forcing a list).